pkg/operator/k8sutil/k8sutil.go

```diff
@@ -63,3 +64,18 @@ func GetK8SVersion(clientset kubernetes.Interface) (*version.Version, error) {
 	}
 	return version.MustParseSemantic(serverVersion.GitVersion), nil
 }
+
+// Hash MD5 hash a given string
+func Hash(s string) string {
+	return fmt.Sprintf("%x", md5.Sum([]byte(s)))
+}
+
+// TruncateNodeName hashes the nodeName in case it would case the name to be longer than 63 characters
+// WARNING If your format and nodeName as a hash, are longer than 63 chars it won't be truncated!
+// Your format alone should only be 31 chars at max because of MD5 hash being 32 chars.
+func TruncateNodeName(format, nodeName string) string {
+	if len(nodeName)+len(fmt.Sprintf(format, "")) > 63 {
```

Comment:
is there a k8s constant we could reference? Or else is there k8s documentation we could mention in the comments about why this is 63?

Reply:
Yep, seem to be this one https://github.com/kubernetes/kubernetes/blob/8d7d7a5e0d4d7e75f5a860574346944b8cc0fc43/staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go#L109